apps/web/pages/event-types/[type].tsx

```diff
@@ -648,7 +648,6 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
   return (
     <div>
       <Shell
-        centered
```

Comment:
for consistency, I've uncentered event-types detail view.

we may as well get rid of the "centered" prop for Shell.